StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/StyleCop.Analyzers.CodeFixes.csproj

```diff
@@ -36,6 +36,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="1.2.1" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.2.1" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
Is this ok?

Reply:
I'd be concerned about the runtime impact of this on VS 2015. It seems like we could use `out` parameters instead pretty easily.

Reply:
Absolutely. I will update.

Reply:
Updated